optaplanner-core/src/main/java/org/optaplanner/core/impl/domain/solution/descriptor/SolutionDescriptor.java

```diff
@@ -88,6 +89,8 @@
 public class SolutionDescriptor<Solution_> {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SolutionDescriptor.class);
+    private static final EntityDescriptor<?> DUMMY_ENTITY_DESCRIPTOR =
```

Comment:
Thinking about a bit more appropriate name: `NONE_ENTITY_DESCRIPTOR` or `NULL_ENTITY_DESCRIPTOR` ?

Reply:
I don't recall formatting it like that. It must have been the auto-formatter, probably because of line length.
I can remove the line break, but if it was the tool, it will just put it right back.